documents/motions/10.2017.0 - Committees and Teams.md

```diff
@@ -32,4 +32,9 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
       3. The WCA Board of Directors shall have the right to withdraw the appointment of a Committee Leader, if the Committee Leader does not comply with the rights and duties of Committee Leader, after advisory consultation by the WCA Ethics Committee.
          1. For the withdrawal of the WCA Ethics Committee Leader, the WCA Board of Directors will ask for advisory consultation to the WCA Disciplinary Committee.
          2. If none of these consultations is possible due to the inexistence of these Committees or because these Committees are not functional, the WCA Board of Directors will proceed without consultation.
-5. All members of the Committee shall communicate with a carbon copy (CC) to all the other members of the Committee.
+5. Each WCA Committee and Team may have one or more Senior Member(s).
+   1. Senior Members are eligible for promotion after they have served at least 1 year in at least one of the Committees.
+      1. If a Member joins a new Team and lefts the old one, the time spent in the old Team shall count for the 1 year period.
+   2. Leaders shall appoint Members for promotion to become Senior Members who are doing and outstanding work for their Team and for the WCA.
```

Comment:
This reads weird to me. It sounds like it's saying that Members who aren't doing outstanding work would be appointed to become Senior Members who are now doing outstanding work?

I would reorder this a bit. How about:

- Leaders shall (will? can? may?) appoint Members who are doing outstanding work to the role of Senior Member.